test/test_cli.py

```diff
@@ -484,6 +484,7 @@ def test_CVSS_score(self, capsys, caplog):
             my_test_filename_pathlib.unlink()
         caplog.clear()
 
+    @pytest.mark.skip(reason="Needs database rebuild. Temporary fix.")  
```

Comment:
```suggestion
    @pytest.mark.skip(reason="Needs database rebuild. Temporary fix.")
```

I think this will fix the black/flake8 issues